packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.js

```diff
@@ -1,10 +1,46 @@
 import {LightningElement, api} from 'lwc';
+import LOCALE from '@salesforce/i18n/locale';
 
 export default class QuanticFacetValue extends LightningElement {
-  /** @type {import("coveo").FacetValue} */
   @api item;
```

Comment:
I think you can do something like this:

`/** @type {import("coveo").FacetValue|import("coveo").DateFacetValue|...} */`

To have a mixed type parameter

Reply:
Yeah but I don't think that really works because then you can't safely reference members that only exist on one of the types. ex: Property 'start' does not exist on type 'FacetValue'. Unless you know how to do existence checks in js that would satisfy tsc...